server/schema.ts

```diff
@@ -266,3 +266,10 @@ export interface IAdminTemplate extends ICommonTemplate {
 		};
 	};
 }
+
+export type DataLog = {
```

Comment:
They're pretty similar (although types are a bit more flexible) but interfaces are a bit better for representing objects because they can be extended and implemented.